Trappist/src/Promact.Trappist.Core/Controllers/QuestionController.cs

```diff
@@ -71,11 +71,11 @@ public QuestionController(IQuestionRepository questionsRepository, UserManager<A
         /// API to get all the Questions
         /// </summary>
         /// <returns>Questions List</returns>
-        [HttpGet]
-        public async Task<IActionResult> GetAllQuestionsAsync()
+        [HttpGet("{numberOfCalls}/{categoryId}/{difficultyLevel}/{searchQuestion?}")]
```

Comment:
if user doesnt search for anything this would be "" and if I dont put question mark then aftaer difficulty level it will be blank and no path will be matching and will give exception